### ISSUE TITLE
Allow standard output format in logger to be configured

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy#214]
As discussed in #192, it would be nice to allow the standard output format to be configured in the same way as the log file format.

**Comment by pllim** [comment]
Is this still necessary? If so, configured how exactly? Hard to follow #192 that happened 8 years ago.

**Comment by bsipocz** [comment]
I wouldn't be against of closing this off. As it seems not much discussion happened since this was opened, neither from the user side, nor from a dev side who is eager to implement it. 
So what about closing it?
